client/js/components/GameView/GameDialogs.vue

```diff
@@ -0,0 +1,189 @@
+<template>
+  <div class="game-dialogs">
```

Comment:
The only major shift here is with this wrapper so we have a singular parent for this component. No styles on it or anything, though.